src/pygbag/filtering.py

```diff
@@ -16,24 +16,24 @@
 /.github
 /.vscode
 /.idea
-/.venv
-/.tox
 /.DS_Store
 /dist
 /build
 /venv
 /ignore
 /static
 /ATTIC
-""".strip().split(
-    "\n"
-)
+""".splitlines()
 
-SKIP_EXT = ["lnk", "pyc", "pyx", "pyd", "pyi", "exe", "bak", "log", "blend", "DS_Store"]
+SKIP_EXT = ["pyc", "pyx", "pyd", "pyi", "exe", "bak", "log", "blend", "DS_Store"]
```

Comment:
why remove "lnk" from exclusion ?

Reply:
I dont know why lnk got removed, I probably deleted it accidently and forgot about it